packages/vuetify/src/components/VDefaultsProvider/VDefaultsProvider.tsx

```diff
@@ -14,19 +14,22 @@ export const VDefaultsProvider = genericComponent(false)({
 
   props: {
     defaults: Object as PropType<DefaultsOptions>,
+    disabled: Boolean,
     reset: [Number, String],
     root: Boolean,
     scoped: Boolean,
   },
 
   setup (props, { slots }) {
-    const { defaults, reset, root, scoped } = toRefs(props)
-
-    provideDefaults(defaults, {
-      reset,
-      root,
-      scoped,
-    })
+    const { defaults, disabled, reset, root, scoped } = toRefs(props)
+
+    if (!disabled.value) {
```

Comment:
Not reactive

Reply:
Nothing in this component is reactive

Reply:
Neither is anything else in this component.

Reply:
defaults, reset, root, scoped are passed to provideDefaults as refs and used in a computed property